app/data/views.py

```diff
@@ -198,6 +202,22 @@ def toddow(self, request):
         return Response(counted)
 
     @list_route(methods=['get'])
+    def recent_counts(self, request):
+        """ Return the recent record counts for 30, 90, 365 days """
+        now = datetime.datetime.now(tz=pytz.timezone(settings.TIME_ZONE))
+        qs = self.get_filtered_queryset(request).filter(occurred_from__lte=now)
+        durations = {
+            'month': 30,
+            'quarter': 90,
+            'year': 365,
+        }
+
+        counts = {label: qs.filter(occurred_from__gte=(now - datetime.timedelta(days=days))).count()
+                  for label, days in durations.items()}
```

Comment:
Hmm, I'm waffling back and forth on this. You can't do the datetime directly in the database because `F` and `Value` objects don't support timedeltas, as far as I've been able to discover. If it was possible, what you'd have to do is pre-compute the 30-, 90- and 365-day cutoff datetimes in Python, and then put those into a `Case` statement. But I'm not sure if that would actually work because I think the "buckets" being passed to the `Case` statement may need to be mutually exclusive in order for the counts to be correct. I'd be happy to poke at this a bit tomorrow if you'd like.